lib/entities/wallet_group.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:cw_core/wallet_info.dart';
+
+class WalletGroup {
+  WalletGroup(this.parentAddress) : wallets = [];
+
+  /// Main identifier for each group, compulsory.
+  final String parentAddress;
+
+  /// Represents the lead wallet of the group (parent or first child)
+  WalletInfo? leadWallet;
```

Comment:
I don't see the need for this, I believe it can be removed, and we can just check if group.wallets.isEmpty

or am I missing something?